pika/exchange_type.py

```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class ExchangeType(Enum) :
+    direct = "Exchange type used for AMQP direct exchanges."
```

Comment:
Please change this enum as follows:

```
direct = 'direct'
fanout = 'fanout'
headers = 'headers
topic = 'topic'
```

Use docstrings to explain what each value is. Then, throughout your pull request, you can remove `.name` from the enum's usage everywhere.

Reply:
If remove `.name` it will fail because with the current usage of Pika `exchange_type` should be a string not a EnumClass .
![image](https://user-images.githubusercontent.com/8179055/73959490-654c2100-492f-11ea-9977-773d4a563f48.png)

Reply:
I suggest changing where it is used to accept either a string or `Enum`